src/test/java/org/folio/rest/domain/FeeFineNoticeContextTest.java

```diff
@@ -66,6 +66,7 @@ public FeeFineNoticeContextTest(Owner owner, Feefine feeFine, Feefineaction feeF
     Feefineaction waivedPartially = createActionWithType("Waived partially");
     Feefineaction transferredFully = createActionWithType("Transferred fully");
     Feefineaction transferredPartially = createActionWithType("Transferred partially");
+    Feefineaction canceledAsError = createActionWithType("Cancelled as error");
```

Comment:
```suggestion
    Feefineaction cancelledAsError = createActionWithType("Cancelled as error");
```

Reply:
This new object does not seem to be used anywhere.

Reply:
Fixed

Reply:
```suggestion
    Feefineaction cancelledAsError = createActionWithType("Cancelled as error");
```